docs/additional-deployment-options/configure_cloudwatch_logs_delivery.md

```diff
@@ -6,12 +6,13 @@ hide_title: true
 
 # Configure API Gateway CloudWatch Logs Delivery
 
-It is possible to deliver CloudWatch API execution and access logs to a cross-account shared AWS::Logs::Destination. An operator does this by adding the key `logToSharedDestination` whose value is a writable log destination to the default level of the Cumulus `config.yml`.
+It is possible to deliver CloudWatch API execution and access logs to a cross-account shared AWS::Logs::Destination. An operator does this by setting the `log_destination_arn` variable whose value is the ARN of a writable log destination to the variables passed to the `cumulus` module.
 
 ```yaml
-default:
-  logToSharedDestination: arn:aws:logs:us-east-1:123456789012:destination:CumulusLogDestination
+log_api_gateway_to_cloudwatch = true
+log_destination_arn           = arn:aws:logDestination
```

Comment:
Is this supposed to be a sample ARN? If so I think the example in the previous version was somewhat clearer

Reply:
According to the doc it can be either a log destination OR a kinesis stream, so I think the previous example is too specific (current one is too, though). I'll try and make it better.